test/simplify_qdq_test.cpp

```diff
@@ -0,0 +1,484 @@
+#include <migraphx/simplify_qdq.hpp>
+#include <migraphx/program.hpp>
+#include <migraphx/ref/target.hpp>
+#include <migraphx/instruction.hpp>
+#include <test.hpp>
+#include <migraphx/make_op.hpp>
+#include <migraphx/dead_code_elimination.hpp>
+#include <migraphx/pass_manager.hpp>
+#include <migraphx/matcher.hpp>
+
+bool is_convolution(const migraphx::instruction& ins) { return ins.name() == "convolution"; }
+bool is_dot(const migraphx::instruction& ins) { return ins.name() == "dot"; }
+
+static migraphx::shape s1{migraphx::shape::int8_type, {1}};
+static migraphx::shape s2{migraphx::shape::int8_type, {1280, 1000}};
+static migraphx::shape s3{migraphx::shape::int8_type, {1000}};
+static migraphx::shape s4{migraphx::shape::int8_type, {1280, 320, 1, 1}};
+static migraphx::shape s5{migraphx::shape::int32_type, {1}};
+static migraphx::shape s6{migraphx::shape::int32_type, {1280}};
+static migraphx::shape s7{migraphx::shape::float_type, {1, 320, 7, 7}};
+static migraphx::shape s8{migraphx::shape::float_type, {1}};
+
+static std::vector<std::vector<float>> floats{{0.00122128},
+                                              {0.168607},
+                                              {0.00260497},
+                                              {0.0235534},
+                                              {0.0235534},
+                                              {0.023622},
+                                              {0.0162265},
+                                              {0.0138147}};
+static std::vector<std::vector<int>> ints{{-35}, {-127}, {-18}, {0}};
+
+TEST_CASE(conv)
+{
+    migraphx::module m1;
+    {
+        auto pa  = m1.add_parameter("pa", s7);
+        auto p3  = m1.add_parameter("p3", s4);
```

Comment:
Can we use better names for the parameters? There is a `p3` and `pa` but no `p1` or `p2`. It would be better name it `input` and `weights`.